vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -148,10 +148,9 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.20.2/windowszip/v1.20.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.0/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0.zip",
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.0_hotfix/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.0_hotfix.zip",
             "https://acs-mirror.azureedge.net/azure-cni/v1.2.2/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.2.zip",
-            "https://acs-mirror.azureedge.net/azure-cni/v1.2.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip"
+            "https://acs-mirror.azureedge.net/azure-cni/v1.2.6/binaries/azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip",
```

Comment:
How do we assure 1.2.0 is not used for existing clusters and should we also remove Linux 1.2.0 and 1.2.0_hotfix cache?

Reply:
Removing cached file does not break functionality but only increase the provisioning time to download cni from the internet. 
In the past, we release the new VHD with the new CNI version change together. Now we are using toggle to update Azure CNI version. They cannot always match. Sometimes cni version is changed before a new VHD is built.
I do not want to remove Linux cached files and leave them to Linux guys to remove it. Linux is still using 1.2.0_hotfix.

Reply:
We keep 3 traditionally if we need to roll back toggles.  so I'm fine with this. Good to check the toggle isn't behind though.